### PR TITLE
[ibexa/cdp] IBX-3795: Added recipe for standalone ibexa/cdp package installation

### PR DESCRIPTION
**Pull request by webhdx**
> Issue: https://issues.ibexa.co/browse/IBX-3795

### CODE REVIEW

**Review by alongosz — APPROVED** [review]

**Review by konradoboza — APPROVED** [review]

File: ibexa/cdp/4.2/config/packages/ibexa_cdp.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+#                account_number: 123456
+#                user_data_streaming:
+#                    # Use Stream ID generated during data import from "stream file" in Data Manager
+#                    stream_id: 00000000-00000000-00000000-00000000
+#                # You can configure multiple activations here. They have to be of type "Ibexa" in Ibexa DXP dashboard.
+#                activations:
+#                    -
+#                        # Client ID and Secret pair is used to authenticate against Webhook endpoint, make sure it's random and secure
+#                        client_id: 123456
+#                        client_secret: abcdef
```
Comment (adamwojs) [inline ibexa/cdp/4.2/config/packages/ibexa_cdp.yaml:15]: client_id and client_secret should be rather an env variable

Comment (webhdx) [reply ibexa/cdp/4.2/config/packages/ibexa_cdp.yaml]: Fixed in [2113526](https://github.com/ibexa/recipes-dev/pull/23/commits/211352660d14c7d194035188dfc3b6fdd24bfe1b)

**Review by adamwojs — APPROVED** [review]

File: ibexa/cdp/4.2/config/packages/ibexa_cdp.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+#ibexa:
+#    system:
+#        default:
+#            cdp:
+#                # Account number can be obtained from User Account settings in Ibexa CDP dashbaord
```
Comment (MagdalenaZuba) [inline ibexa/cdp/4.2/config/packages/ibexa_cdp.yaml:5]:
```suggestion
#                # Account number can be obtained from User Account settings in Ibexa CDP dashboard
```

Comment (MagdalenaZuba) [reply ibexa/cdp/4.2/config/packages/ibexa_cdp.yaml:5]: And it is not User Account settings but Accounts/Company accounts settings, you cannot find it under your profile settings.

**Review by Nattfarinn — APPROVED** [review]

**Review by mikadamczyk — APPROVED** [review]

**Pull request merged**